aw_watcher_window/main.py

```diff
@@ -11,22 +10,21 @@
 from aw_client import ActivityWatchClient
 
 from .lib import get_current_window
-from .config import load_config
+from .config import parse_args
 
 logger = logging.getLogger(__name__)
 
+# run with LOG_LEVEL=DEBUG
+log_level = os.environ.get('LOG_LEVEL')
+if log_level:
+    logger.setLevel(logging.__getattribute__(log_level.upper()))
 
 def main():
-    # Read settings from config
-    config = load_config()
-    args = parse_args(
-        default_poll_time=config.getfloat("poll_time"),
-        default_exclude_title=config.getboolean("exclude_title"),
-    )
-
     if sys.platform.startswith("linux") and ("DISPLAY" not in os.environ or not os.environ["DISPLAY"]):
         raise Exception("DISPLAY environment variable not set")
 
+    args = parse_args()
```

Comment:
This should happen before the check above in order for the CLI to behave properly in the absence of an X server.

Reply:
Ah, got it. Will move!